server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/view/cassandra/configuration/ConfigurationAggregate.java

```diff
@@ -54,6 +54,9 @@ private void validateConfiguration(CassandraMailQueueViewConfiguration configura
             Preconditions.checkNotNull(configurationUpdate);
 
             maybeConfiguration.ifPresent(currentConfiguration -> {
+                Preconditions.checkArgument(!currentConfiguration.equals(configurationUpdate),
```

Comment:
that's an interesting case. the aggregate should accept the input and generate no events.